provider/core/Cargo.toml

```diff
@@ -35,6 +35,8 @@ extra_features = ["serde"]
 all-features = true
 
 [features]
+default = ["sync"]
```

Comment:
Discussion: I think `"sync"` should not be default.

Reply:
agree